packages/rpm/hazelcast.spec

```diff
@@ -70,10 +70,6 @@ chown -R hazelcast:hazelcast %{_prefix}/lib/hazelcast/
 printf "\n\nHazelcast is successfully installed to '%{_prefix}/lib/hazelcast/'\n"
 hz --help
 
-%preun
-%systemd_preun %{name}.service
-
-
 %preun
 %systemd_postun %{name}.service
```

Comment:
`%preun` is calling `%systemd_postun`. Maybe it should be `%postun` instead and we should restore `%preun` above?